docs/manual/_static/copybutton.js

```diff
@@ -0,0 +1,248 @@
+// Localization support
+const messages = {
+  'en': {
+    'copy': 'Copy',
+    'copy_to_clipboard': 'Copy to clipboard',
+    'copy_success': 'Copied!',
+    'copy_failure': 'Failed to copy',
+  },
+  'es' : {
```

Comment:
Are we supporting the other languages somewhere else? Do we need this?

Reply:
Thanks, see below.